src/lib/assetFinancials.ts

```
@@ -0,0 +1,233 @@
+    const partialFirstMonthFraction = (daysInAcqMonth - acqDay + 1) / daysInAcqMonth;
+
+    const targetYear = asOfDate.getFullYear();
+    const targetMonth = asOfDate.getMonth();
+
+    const totalRawMonths = (targetYear - acqYear) * 12 + (targetMonth - acqMonth);
+
+    if (totalRawMonths < 0) {
+      return asset.acquisition_cost;
+    }
+
+    const elapsedMonths = totalRawMonths === 0 ? partialFirstMonthFraction : totalRawMonths + partialFirstMonthFraction;
```

ESLint

[error] Insert `⏎·····`

(prettier/prettier)